manticore/__main__.py

```diff
@@ -93,20 +111,40 @@ def positive(value):
 
 
 def ethereum_cli(args):
-    from .ethereum import ManticoreEVM, IntegerOverflow, UninitializedStorage, UninitializedMemory
+    from .ethereum import ManticoreEVM, DetectInvalid, DetectIntegerOverflow, DetectUninitializedStorage, DetectUninitializedMemory, FilterFunctions
     log.init_logging()
 
     m = ManticoreEVM(procs=args.procs)
 
-    ################ Default? Detectors #######################
-    m.register_detector(IntegerOverflow())
-    m.register_detector(UninitializedStorage())
-    m.register_detector(UninitializedMemory())
+    if args.detect_all:
```

Comment:
Is setting `args` attributes an accepted way of doing this? instead of creating aliases or just checking for `detect_all` flags below?